lib/ex_aws/dynamo.ex

```diff
@@ -364,6 +406,25 @@ defmodule ExAws.Dynamo do
   defp do_convert_billing_mode(attributes, value) when is_list(attributes),
     do: Keyword.replace!(attributes, :billing_mode, value)
 
+  defp add_table_opt({:stream_enabled, enabled}, data) do
+    data
+    |> ensure_entry("StreamSpecification")
+    |> put_in(["StreamSpecification", "StreamEnabled"], enabled)
+  end
+
+  defp add_table_opt({:stream_view_type, type}, data) do
+    data
+    |> ensure_entry("StreamSpecification")
+    |> put_in(["StreamSpecification", "StreamViewType"], type |> to_string() |> String.upcase())
```

Comment:
I went back and forth on that a bit, but ended up sticking with what was already there. I don't have any real objection to doing it, but it may look a bit out of place when all the other string field names are inline.

Reply:
Cool man.